.mergify.yaml

```diff
@@ -0,0 +1,23 @@
+pull_request_rules:
+  - name: Merge approved and green PRs with merge-when-green label
+    conditions:
+      - "#approved-reviews-by>=1"
+      - status-success=Travis CI - Pull Request
```

Comment:
Why travis? we us GH now

Reply:
yup, will fix. Actually this whole section i am removing as it's ALL PRs, and I just want Dependabot